data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py

```diff
@@ -36,6 +36,9 @@
       WHEN cssf.cope_month = 'may' THEN 2100000002
       WHEN cssf.cope_month = 'june' THEN 2100000003
       WHEN cssf.cope_month = 'july' THEN 2100000004
+      WHEN cssf.cope_month = 'november' THEN 2100000005
+      WHEN cssf.cope_month = 'december' THEN 2100000006
+      WHEN cssf.cope_month = 'february' THEN 2100000007
```

Comment:
Cope month values should be `nov`, `dec`, and `feb`.

Reply:
Thanks for catching this!